output/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Instead of adding this file, add this directory in the root directory's .gitignore.

Reply:
git doesn't create empty folders when we initially clone the repo. So, this is one of the work-arounds:
https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

Actually, as the next step, what do you think of removing "output-dir" parameter from the script? So it always creates the csv files in "output" folder.

The problem is that it tries to create the files at the end of the script and if the "output-dir" is not there, it throws an exception and the whole execution fails:

```shell
Traceback (most recent call last):
  File "C:\Users\Serkan\AppData\Local\Programs\Python\Python36-32\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\Serkan\AppData\Local\Programs\Python\Python36-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Development\Projects\GitHub\Temp\criticality_score\criticality_score\generate.py", line 163, in <module>
    main()
  File "D:\Development\Projects\GitHub\Temp\criticality_score\criticality_score\generate.py", line 152, in main
    with open(output_filename, 'w') as file_handle:
FileNotFoundError: [Errno 2] No such file or directory: 'output\\all_top_1.csv'
```

Reply:
Your patch is fine then, merging.